test/unit/schema-builder/postgres.js

```diff
@@ -1168,6 +1181,19 @@ describe('PostgreSQL SchemaBuilder', function () {
     );
   });
 
+  it('adding datetime with timezone', () => {
+    tableSql = client
+      .schemaBuilder()
+      .table('users', (table) => {
+        table.datetime('foo', false);
```

Comment:
Maybe I'm missing some context but wouldn't passing `false` here result in timestamp WITHOUT timezone? Why does it seem inverted?

Reply:
- It's Because this is the original design 
![image](https://user-images.githubusercontent.com/48720253/126087966-a6df4f1a-f349-40d4-9c70-fd27d592760b.png)

- And this is the original test for timestamp() which do the same thing as datetime():
![image](https://user-images.githubusercontent.com/48720253/126087750-2a241cd5-e9c0-44b2-b734-4252a9346782.png)

I did changed it to withTz so it does not seem inverted anymore. But as @kibertoad said, it will cause breaking change so i  just add the bug fix and sometest without changing the parameter. Maybe we could request this as feature for future update because i agree that it is not intuiitive. 
Also it is only mentioned in the documentation that you can use ```optiton: {useTz: boolean, precision: number}``` so i don't know if many people create timezone like that